Skip CpuTests.test_constructors_pin_memory_tensor_{inline,constant}_cpu failing with
> AssertionError: False is not true

Regression from https://github.com/pytorch/pytorch/pull/183977
See https://github.com/pytorch/pytorch/issues/197211

Author: Alexander Grund (TU Dresden)

diff --git a/test/inductor/test_compile_subprocess.py b/test/inductor/test_compile_subprocess.py
--- a/test/inductor/test_compile_subprocess.py
+++ b/test/inductor/test_compile_subprocess.py
@@ -81,6 +81,8 @@ test_failures = {
     "test_split_overload_packet_lowering": TestFailure(
         ("cpu", "cuda", "xpu"), is_skip=True
     ),
+    "test_constructors_pin_memory_tensor_constant": TestFailure("cpu", is_skip=True),
+    "test_constructors_pin_memory_tensor_inline": TestFailure("cpu", is_skip=True),
 }
 
 if TEST_WITH_ROCM and not torch.cuda.has_magma:
